openpower/package/hcode/hcode.mk

```diff
@@ -0,0 +1,42 @@
+################################################################################
+#
+# HCODE
+#
+################################################################################
+
+#$TODO fix this..
+HCODE_VERSION ?= 1ef7a601489b66d5e7ded9905de6cede4696d93a
+HCODE_SITE ?= $(call github,open-power,hcode,$(HCODE_VERSION))
+
+HCODE_LICENSE = Apache-2.0
```

Comment:
please check that `op-build legal-info` produces the correct result and finds the license file.